code/modules/surgery/organic_steps.dm

```diff
@@ -142,7 +162,12 @@
 //drill bone
 /datum/surgery_step/drill
 	name = "drill bone"
-	implements = list(TOOL_DRILL = 100, /obj/item/screwdriver/power = 80, /obj/item/pickaxe/drill = 60, TOOL_SCREWDRIVER = 25, /obj/item/kitchen/spoon = 20)
+	implements = list(
+		TOOL_DRILL = 100,
+		/obj/item/screwdriver/power = 30,
+		/obj/item/pickaxe/drill = 25,
+		TOOL_SCREWDRIVER = 20,
+		/obj/item/kitchen/spoon = 10) //spoon? really? this should be awful.
```

Comment:
make it awful then

Reply:
1/10 is pretty awful. I might just take it out.

Reply:
i made it worse